src/composables/search.js

```diff
@@ -28,6 +33,18 @@ export const useSearchStore = defineStore('search', () => {
     return await $fetch(`/api/search/${id}`)
   }
 
+  async function isNameAvailable(name) {
+    try {
+      await axios.get(`${MIDDLEWARE_URL}/names/${name}`)
+      return true
+    } catch (error) {
+      if (error.response.status === 404) {
+        return false
+      }
+      return null
```

Comment:
Do some logging so as not to swallow the error. I don't think `null` is useful, propose to return `false` instead

Reply:
The error is signalized to user thru UI. But you are right, changed to false.

Reply:
Make sense